inputs/simple-list.vue

```diff
@@ -152,6 +153,15 @@
       };
     },
     computed: {
+      autocompleteOpts() {
+        let opts = this.args.autocomplete;
+
+        if (_.has(this.args, 'modifyList') && _.has(this.args.modifyList, 'allowDestroy')) {
```

Comment:
Wait, I inferred from the docs that `modifyList` was a boolean, but that doesn't seem the case? It's an object? What other properties would go in that object?

Reply:
It's an object. I was going to dump `allowCreate` in there. Going to work on that feature in another PR.